dev/com.ibm.ws.concurrent_fat_reactive/publish/servers/com.ibm.ws.concurrent.fat.reactive.ReactiveServer/bootstrap.properties

```diff
@@ -12,3 +12,6 @@
 ###############################################################################
 bootstrap.include=../testports.properties
 com.ibm.ws.logging.trace.specification=*=info:concurrent=all:context=all:concurrencyPolicy=all
+
+#TODO remove this if we still need to run with java security on Java 21+
```

Comment:
You can remove the TODO if you want. We do not need to run with Java 2 security for Jakarta EE 11 features.

Reply:
There's one of these in a jakarta data bucket as well, so I'll add both to my next pr